datasafe-simple-adapter/datasafe-simple-adapter-impl/pom.xml

```diff
@@ -73,6 +73,12 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Do you need to change something in datasafe-simple-adapter at all?

Reply:
No, I am going to remove my changes from datasafe-simple-adapter